stdlib/public/core/UnsafePointer.swift

```diff
@@ -327,12 +398,14 @@ public struct UnsafePointer<Pointee>: _Pointer {
     defer { Builtin.rebindMemory(_rawValue, binding) }
     return try body(.init(_rawValue))
   }
+}
 
+extension UnsafePointer {
   // This unavailable implementation uses the expected mangled name
   // of `withMemoryRebound<T, Result>(to:capacity:_:)`, and provides
   // an entry point for any binary linked against the stdlib binary
   // for Swift 5.6 and older.
-  @available(*, unavailable)
+  @available(swift, obsoleted: 5.11)
```

Comment:
Would it be worth putting this in `LegacyABI.swift` instead to clear up this file a little? (You don't have to, just adding a suggestion)

Reply:
Yes, but let's wait for the dust to settle a bit.